repos/system_upgrade/common/actors/targetuserspacecreator/libraries/userspacegen.py

```diff
@@ -681,7 +698,7 @@ def _get_product_certificate_path():
     :return: The path to the product certificate or None on non-RHEL systems
```

Comment:
docstrings need to reflect the code changes too :)

```suggestion
   Product certificates are only used for RHEL. Returns None if the target distro is not RHEL.

   :return: The path to the product certificate or None  if target distro is not RHEL
```